### PR TITLE
removed deprecated egress messages variants

### PR DESCRIPTION
**Pull request by 35359595**
Removed:

-    EnclaveAccountDump
-   EnclaveSnapshot

Based on refactoring for Orderbook's #179

### CODE REVIEW

File: src/egress.rs [hunk 1]
```diff
@@ -9,23 +9,7 @@ use scale_info::TypeInfo;
...
 // #[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
-#[scale_info(skip_type_params(SnapshotAccLimit, WithdrawalLimit))]
-pub enum EgressMessages<
-    AccountId: Ord,
-    Balance: Zero + Clone,
-    ProxyLimit: Get<u32>,
-    WithdrawalLimit: Get<u32>,
-    AssetsLimit: Get<u32>,
-> {
-    EnclaveAccountDump(
```
Comment (serhii-temchenko) [inline src/egress.rs:20]: We have consumers of this variant [here](https://github.com/Polkadex-Substrate/orderbook/blob/master/enclave/src/import_blocks.rs#L117) and [here](https://github.com/Polkadex-Substrate/orderbook/blob/master/relayer/src/import.rs#L305). I assume this should be refactored as well?

File: src/egress.rs [hunk 2]
```diff
@@ -9,23 +9,7 @@ use scale_info::TypeInfo;
...
-        EnclaveAccountInfoDump<AccountId, Balance, ProxyLimit>,
-        Signature,
-    ),
-    EnclaveSnapshot(
```
Comment (serhii-temchenko) [inline src/egress.rs:24]: We have consumers of this variant [here](https://github.com/Polkadex-Substrate/orderbook/blob/master/enclave/src/import_blocks.rs#L112) and [here](https://github.com/Polkadex-Substrate/orderbook/blob/master/relayer/src/import.rs#L334). I assume this should be refactored as well?

File: src/egress.rs (hunk 1 above)
Comment (35359595) [reply src/egress.rs:20]: will be removed as part of https://github.com/Polkadex-Substrate/orderbook/issues/179

File: src/egress.rs (hunk 2 above)
Comment (35359595) [reply src/egress.rs:24]: will be removed as part of https://github.com/Polkadex-Substrate/orderbook/issues/179

**Review by serhii-temchenko — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Review by Gauthamastro — APPROVED** [review]

**Pull request merged**